distribution/docs/src/main/resources/content/_configuring/decanter-services.adoc

```diff
@@ -1,15 +1,16 @@
-:title: Configuring Decanter Services
-:type: configuringConfigFile
+:title: Configuring Alerts
+:type: configuration
 :status: published
-:summary: Configuring Decanter services by configuration file.
-:order: 14
+:parent: Miscellaneous Configurations
+:summary: Configuring alerts through Decanter services.
+:order: 02
 
-==== Configuring Decanter Services By Configuration file
+== {title}
 
-By default, ${branding} uses two services provided by Decanter that can be configured by configuration file.
+By default, ${branding} uses two services provided by Decanter for alerts that can be configured by configuration file.
```

Comment:
Should we call this `Karaf Decanter`  in each of the places it is used?

Reply:
Probably fixed